index.html

```diff
@@ -156,11 +161,78 @@ <h2 class="text-gray-900 title-font text-lg font-medium">Fogg Wrist Watch</h2>
 
           <!--Footer-->
           <footer class="text-gray-700 body-font">
-            <div class="bg-gray-200">
+            <!-- <div class="bg-gray-200">
               <div class="container mx-auto py-4 px-5 flex flex-wrap flex-col sm:flex-row">
                 <p class="text-gray-500 text-sm text-center sm:text-left">&copy; 2020 ShopOnline
                 </p>
               </div>
+            </div> -->
+            <div class="footer">
+              <div class="heading">
+                <a class="flex title-font font-medium items-center text-gray-900 mb-4 md:mb-0">
+                  <img class="ml-3 text-xl" src="https://cdn3.iconfinder.com/data/icons/small/512/buy_cart_ecommerce_shopping_online_store-512.png" style="height: 50px;"></img>
+                  <span class="ml-3 text-xl"><text  style="font-size: 20px; font-family:'Montserrat', sans-serif;color: #000000;">ShopOnline</text></span>  
+              </a>
+              </div>
+              <div class="content">
```

Comment:
Good that you've added a lot. But unfortunately these aren't the services the platform offers. Maybe you can modify content on page that aligns with the services the platform offers

Reply:
okay sure , but can i get a list of all the links that you want to have in the footer , so that I don't miss out anything.

Reply:
you can add GitHub repo link, my twitter id @Sandeeprajakris and sudhay2001@gmail.com